data_steward/cdr_cleaner/cleaning_rules/validate_missing_participant.py

```diff
@@ -0,0 +1,247 @@
+import logging
+import bq_utils
+import resources
+import oauth2client
+import googleapiclient
+from validation.participants import readers
+from cdr_cleaner.cleaning_rules import sandbox_and_remove_pids as remove_pids
+from constants.validation.participants.identity_match import (PERSON_ID_FIELD,
+                                                              FIRST_NAME_FIELD,
+                                                              LAST_NAME_FIELD,
+                                                              BIRTH_DATE_FIELD)
+from constants.validation.participants.writers import ALGORITHM_FIELD
+from common import PARTICIPANT_MATCH
+
+IDENTITY_MATCH = 'identity_match'
+
+LOGGER = logging.getLogger(__name__)
+
+TICKET_NUMBER = 'DC-468'
+
+NUM_OF_MISSING_KEY_FIELDS = 2
+NUM_OF_MISSING_ALL_FIELDS = 4
+
+KEY_FIELDS = [FIRST_NAME_FIELD, LAST_NAME_FIELD, BIRTH_DATE_FIELD]
+
+IDENTITY_MATCH_EXCLUDED_FIELD = [PERSON_ID_FIELD, ALGORITHM_FIELD]
+
+CAST_MISSING_COLUMN = "CAST({column} <> 'match' AS int64)"
+
+CRITERION_COLUMN_TEMPLATE = "({column_expr}) >= {num_of_missing}"
+
+SELECT_NON_MATCH_PARTICIPANTS_QUERY = """
+WITH non_match_participants AS
+(
+SELECT 
+    *,
+    ({criterion_one_expr}) AS criterion_one,
+    ({criterion_two_expr}) AS criterion_two
+FROM `{project_id}.{validation_dataset_id}.{identity_match_table}`
+)
+
+SELECT
+    person_id
+FROM non_match_participants
+WHERE criterion_one IS TRUE OR criterion_two IS TRUE
```

Comment:
nit: It would be more descriptive to use variable names like key_fields_criteria and all_fields_criteria instead of one and two, but this is not a blocker.

Reply:
This sounds like a good idea.